docs/advanced-features/instrumentation.md

```diff
@@ -0,0 +1,37 @@
+---
+description: Learn how instrument your Next.js app.
+---
+
+> **Note**: This feature is experimental, you need to explicitly opt-in by providing `experimental.instrumentationHook = true;` in your `next.config.js`.
+
+# `instrumentation.ts`
+
+You can export `register` function from this file and we will be called whenever we bootstrap a new server instance.
+When deployed it will be called continuously on each coldboot (but exactly once in each environment).
```

Comment:
```suggestion
When your `register` function is deployed, it will be called continuously on each cold boot (but exactly once in each environment).
```

Is the `register` function what you're referring to when you say "it"? I totally understand if it seems redundant to keep being explicit here, but I do feel it helps when the subject is so technical. One of our docs goals is to make everything as approachable to juniors as possible